### ISSUE TITLE
Autodetect value format [$20 awarded]

### ISSUE THREAD

**Issue by uglide** [redis/RedisDesktopManager#3310]
Autodetect value format using following rule:

```
if binary:
  if valid_msgpack:
    return msg_pack
  else:
    return HEX
else:
  if valid_json:
    return json
  elseif valid_php:
    return php
  else:
    return plain_text
```
##

**Comment by uglide** [comment]
Implemented in 0.8.0-RC5 (will be released soon)

**Comment by doberloh** [comment]
This is not working as intended. In the last build plain text seemed to be the default, now it loads up all of my values in HEX, I cant switch the mode back to plain text, and everything is unreadable.

**Comment by uglide** [comment]
@doberloh Fix will be shipped in 0.8.2
